content/api/3-postman-collection.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Postman Collection"
+metaTitle: "Postman Collection"
+metaDescription: "Spectro Cloud API Postman Collection"
+icon: ""
+hideToC: true
+fullWidth: false
+hiddenFromNav: false
+hideToCSidebar: true
+---
+import {Intro, IntroButtons} from "shared/components"
+import InfoBox from "shared/components/InfoBox"
+
+<Intro>
+
+# Postman Collection
+
+</Intro>
+
+Spectro Cloud provides a Postman collection for your convenience to help you interact with the Spectro Cloud API.
+
+To interact with the API, you will need the following:
```

Comment:
We can lump the following paragraph + info box into a prerequisites section. We also should add a bullet specifying Postman App vX.X.X or greater.